migrations/6_limitorders.js

```diff
@@ -0,0 +1,54 @@
+/*
+
+    Copyright 2019 dYdX Trading Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License";
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+*/
+
+const {
+  isDevNetwork,
+  getChainId,
+} = require('./helpers');
+
+// ============ Contracts ============
+
+const SoloMargin = artifacts.require('SoloMargin');
+const TestSoloMargin = artifacts.require('TestSoloMargin');
+const LimitOrders = artifacts.require('LimitOrders');
+
+// ============ Main Migration ============
+
+const migration = async (deployer, network) => {
+  const soloMargin = await getSoloMargin(network);
+  await deployer.deploy(
+    LimitOrders,
+    soloMargin.address,
+    getChainId(network),
+  );
+  await soloMargin.ownerSetGlobalOperator(
```

Comment:
is this just for tests? do you use these migrations on prod? or how do you migrate on prod

Reply:
Just for tests yeah, our migrations are a bit messed up now and we basically just do it in a bespoke manner for any contract release and then we verify it on-chain